app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
+    margin-left: -70px ;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Declaration should not have a space before the terminating semicolon